sys/include/net/gnrc/netif/internal.h

```diff
@@ -349,16 +375,25 @@ bool gnrc_netif_is_6ln(const gnrc_netif_t *netif);
  * @return  true, if the interface represents a 6LR
  * @return  false, if the interface does not represent a 6LR
  */
+#if (GNRC_IPV6_NIB_6LR && \
```

Comment:
I think this should be `GNRC_IPV6_NIB_CONF_6LR`. This might explain why neighbor address registration is broken.

Reply:
I think I might have been drunk, when I implemented that 😅